services/src/main/scala/cromwell/pipeline/service/InputNodeService.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.pipeline.service
+
+import cats.implicits._
+import cromwell.pipeline.datastorage.dao.repository.DocumentRepository
+import cromwell.pipeline.datastorage.dto.InputNode
+import org.mongodb.scala.Completed
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class InputNodeService(repository: DocumentRepository)(implicit ec: ExecutionContext) {
+  import InputNode._
+
+  def addNode(node: InputNode): Future[String] =
+    repository.addOne(toDocument(node)).map((status: Completed) => status.toString())
```

Comment:
Do we need a type here map((status: Completed) => status.toString())
I think it could be just like this: map(_.toString)

Reply:
You are right, thx